src/config.rs

```diff
@@ -453,6 +461,38 @@ impl FromStr for Environment {
     }
 }
 
+// -----------------------------------------------------------------------------
+// Enum: Stratus Mode
+// -----------------------------------------------------------------------------
+#[derive(DebugAsJson, strum::Display, strum::VariantNames, Clone, Copy, Parser, serde::Serialize)]
+pub enum StratusMode {
```

Comment:
Modes we want are:
* `Leader` : executes transactions and produces blocks locally in automine or interval mining mode.
* `Follower`: forwards transactions to the leader and replicate leader state using importer-online.

Leader will never use importer-online.
Follower should always have importer-online configured.